### PR TITLE
Advertise some more compiler plugins

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

**Comment by kubukoz** [comment]
OK, I think I'm good with these changes. @SethTisue can you approve me for workflows?

File: _overviews/plugins/index.md [hunk 1]
```diff
@@ -42,18 +42,17 @@ include:
 - Alternate compiler back ends such as [Scala.js](https://www.scala-js.org), [Scala Native](http://scala-native.org), and
   [Fortify SCA for Scala](https://developer.lightbend.com/docs/fortify/current/).
 - Linters such as [Wartremover](https://www.wartremover.org) and [Scapegoat](https://github.com/sksamuel/scapegoat).
-- Plugins that support reformatting and other changes
-  to source code, such as [scalafix](https://scalacenter.github.io/scalafix/) and [scalafmt](https://scalameta.org/scalafmt/) (which are
-  built on the [semanticdb](https://scalameta.org/docs/semanticdb/guide.html) and [scalahost](https://github.com/Duhemm/scalahost) compiler plugins).
```
Comment (SethTisue) [inline _overviews/plugins/index.md:47]: did you mean to remove these lines?

Comment (kubukoz) [reply _overviews/plugins/index.md:47]: Yeah, see this thread: https://github.com/scala/docs.scala-lang/pull/2011#discussion_r630107019

Comment (kubukoz) [reply _overviews/plugins/index.md:47]: actually, the commit explains more too: https://github.com/scala/docs.scala-lang/pull/2011/commits/9aaff97197d191266ceb479c4273ca6b06b86492

>  scalafix/scalafmt aren't compiler plugins, scalahost isn't maintained.

File: _overviews/plugins/index.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ include:
 - Alternate compiler back ends such as [Scala.js](https://www.scala-js.org), [Scala Native](http://scala-native.org), and
   [Fortify SCA for Scala](https://developer.lightbend.com/docs/fortify/current/).
 - Linters such as [Wartremover](https://www.wartremover.org) and [Scapegoat](https://github.com/sksamuel/scapegoat).
+<!-- is this even accurate? scalafmt is not a _compiler_ plugin I think, and scalafix relies on the semanticdb plugin? -->
```
[the remark this answers is not in the document]
Comment (SethTisue) [reply _overviews/plugins/index.md]: Removing scalafmt is fine (I had thought it was semanticdb-based, oops!).

Removing scalahost is fine, too.

But it was never my intention to imply, through poor phrasing, that scalafix is a compiler plugin. I wanted to include it because it's so important and because its existence is enabled by semanticdb-scalac, which is a compiler plugin. Not everyone knows what semanticdb is, but everybody knows about scalafix.

Wdyt about the following replacement phrasing:

> Plugins that enable tooling. One such plugin is semanticdb-scalac, which enables [scalafix](https://scalacenter.github.io/scalafix/) (a well-known refactoring and listing tool) to do its work.

[the remark this answers is not in the document]
Comment (kubukoz) [reply _overviews/plugins/index.md]: Semanticdb is currently mentioned next to Paradise, but I guess I can rephrase it to mention scalafix explicitly ;)

**Comment by kubukoz** [comment]
Took me some time... @SethTisue when you have a moment :)

**Comment by SethTisue** [comment]
Thank you!
